compliance-web/src/components/App/Complaints/ComplaintFormUtils.ts

```diff
@@ -77,26 +77,6 @@ export const ComplaintFormSchema = yup.object().shape({
   }),
   amendmentNumber: yup.string().nullable(),
   amendmentConditionNumber: yup.string().nullable(),
-  description: yup.string().when("requirementSource", {
```

Comment:
we need to keep these fields in the schema, just remove the restriction only, make it similar to amendmentNumber/amendmentConditionNumber